app/views/moments/show.html.erb

```diff
@@ -1,5 +1,7 @@
 <% title @moment.name %>
 <%= render :partial => '/moments/info', locals: { data: @moment } %>
+<%= button_to "Secret Share", secret_shares_path(moment: @moment.name) %>
```

Comment:
This should be i18n key. You'll need to also translate this to Spanish and Portuguese. For Spanish, you can ask in the #es-translations and for Portuguese, #br-translations